app/javascript/store/index.js

```diff
@@ -5,6 +5,7 @@ import profile from './modules/profile';
 import scores from './modules/scores';
 import statistics from './modules/statistics';
 import admin from './modules/admin';
+import froggo_team from './modules/froggo_team';
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Identifier 'froggo_team' is not in camel case  camelcase

Reply:
y acá lo mismo con el nombre de la carpeta :(